app/src/main/java/at/roteskreuz/stopcorona/model/repositories/InfectionMessengerRepository.kt

```diff
@@ -141,11 +144,7 @@ class InfectionMessengerRepositoryImpl(
             Timber.e(IllegalArgumentException("processing of token ${token} when we expect to process ${lastScheduledToken}"))
         }
 
-        if (token.startsWith(BATCH)) {
-            processKeysBasedOnBatchToken(token)
-        } else {
-            processDailyTokens(token)
-        }
+        processKeysBasedOnBatchToken(token)
```

Comment:
review

Reply:
@falkorichter did a review and integrated it into the existing processing